data/hai/hai reveal 0 prologue.txt

```diff
@@ -0,0 +1,734 @@
+			`	Sayari chuckles. "So much on that human's mind. I have not seen him take a break since he has arrived. It is as if he is more stress about this situation than any Hai."`
+				decline
+
+
+
+mission "Hai Reveal: Meet the Team: Xilin"
+	name "New Arrivals"
+	description "Travel to <stopovers> with Xilin to interview new arrivals to Hai space to determine how they arrived and what they think about the Hai. Return to <destination> when finished."
+	source "Hai-home"
+	stopover "Allhome"
+	passengers 2
+	blocked "You will need <capacity> in order to take this mission."
+	to offer
+		has "hr: meet the team"
+		has "Hai Reveal: Meet the Team: Alondo: done"
+	on offer
+		log "People" "Xilin Yang" `Deputy Minister Yang works in the Foreign Affairs department of the Republic government. He is a pleasant and astute man. Handling the Hai secrecy crisis has become his job while the majority of the foreign ministry remains engaged with the fallout from the Free Worlds and recent peace.`
+		conversation
+			`Sayari follows you out to the landing pads to meet the presenter. "Captain <last>," he says while grasping your hand in a firm handshake. "Xilin Yang, Republic Deputy Minister of Foreign Affairs. I apologize for not introducing myself sooner. It's an honor to meet a hero of the Pug crisis such as yourself."`
+			choice
+				`	"It was hard-fought, but it was necessary."`
+				`	"Thanks, but it wasn't as dramatic as they made it out to be. I'm just happy to have helped where I could."`
+					goto humble
+			`	"You're every inch what I expected," he assures you.`
+				goto mission
+			label humble
+			`	For a moment his expression falters, and then he seems almost relieved. "You may be a better person than I expected to meet.`
+			label mission
+			`	"Now, about that task. I mentioned during our briefing that we need to determine how much of a threat these new arrivals might be. Not a threat in the sense that they will become violent, but in the sense that they could exacerbate the situation, such as by returning to our space and making a big stink about what they've found here. The best place to go for this would be <stopovers>."`
+			`	"I would like to follow and observe," says Sayari.`
+			`	"You're free to come," says Xilin. "Let me pack a few items into your ship, Captain <last>, and we can be on our way."`
+				accept
+	on stopover
+		conversation
+			`It doesn't take long after landing on <planet stopovers> to find new arrivals. Every so often a human ship will land and its crew will disembark while gawking at the spaceport, making it very obvious that they're newcomers.`
+			`	Xilin records information about each captain, including their pilot's license, ship registration, how exactly they found their way to the Hai, and more. One of the more concerning events is when a Star Queen full of passengers lands in the spaceport.`
```

Comment:
1 ship per day would be at most about 30,000 humans in a year, and probably closer to 3,000 (most ships are towards the smaller end), not all of whom will be there to stay. That wouldn't be that significant given that the population of Hai space is on the order of tens of billions.

Reply:
Could have used fuel pods instead of ramscoops, they have the advantages of being cheaper, smaller (if you already had 400 fuel and only need one more jump), and not requiring extra time sitting around near the star to refuel.

Reply:
> 1 ship per day would be at most about 30,000 humans in a year, and probably closer to 3,000 (most ships are towards the smaller end), not all of whom will be there to stay. That wouldn't be that significant given that the population of Hai space is on the order of tens of billions. Even assuming the average ship coming through the wormhole carries about 100 people, all of whom want to stay in Hai space, humans would still only be less than a tenth of a percent of the population after 300 years.
> ...
> With the same assumptions, an average of two ships in an hour would mean about a 1% increase in the total population of Hai space over 60 years, which could be sufficient to maybe be problematic if the Hai have lost the ability to handle significant population growth as a result of their millennia of stagnation.


Arguably the issue is not the total Hai population, but the population of Allhome specifically, since the availability of services to receive people is concentrated there where basically ever human ever lands for the first time. Also, unexpected immigration tends to cause strain at like 0.005-0.01%...

> I do prefer the phrasing of your version though, so maybe keep the other changes, but change `in the past day` to `in the last hour`, or maybe `in the last hour alone`? 

I would rather double the numbers per day rather than change the interval.

Reply:
Considering we're seeing two more come in just in the time of this conversation, I think having two in the past day before that is fine. The acceleration of doubling that is already right there in the dialogue.

Reply:
Went a different route.